### ISSUE TITLE
Graphics Revamp

### ISSUE THREAD

**Issue by blackearth2014** [mschack/VCV-Rack-Plugins#27]
MsHack,

Some guys from the Facebook VCVRack resurfaced your graphics for your plugins. They wondered if it will be okay and I told them that I will let you know. I don't care for this design resurface, but wanted to make you aware of it. See attachement.

![graphics_update](https://user-images.githubusercontent.com/7956901/33504601-0e5280b2-d6b6-11e7-8ad1-755332aba244.jpg)

**Comment by mschack** [comment]
Yeah sure that's fine.  I pushed my colours as a bit of an experiment.  I personally don't like everything black and white but if that's what they prefer I have no issues with it.

**Comment by blackearth2014** [comment]
I like them the way there are. They truly stand out and you can't mistake them for some other modules.

**Comment by blackearth2014** [comment]
By any chance are you thinking of entering the sample playback field. There are others modules available but they all have their quirks. I think that you sequencers can use sampler companion. Yiu can also charge based on the features you put in it.

Here is a youtube playlist of tracks that I have done using your modules thus far at https://www.youtube.com/watch?v=6qhACsRPpHg&list=PLKq_Gnn6tV4qxAcLmio5-v7xY1PKQZocm.

**Comment by mschack** [comment]
Well thanks!  Looking at them now they seem to lack inspiration...

**Comment by blackearth2014** [comment]
Cool Breeze
